setup.py

```diff
@@ -25,7 +25,8 @@
     README = readme_file.read()
 
 install_requirements = [
-    "sceptre>=2.7"
+    "sceptre>=2.7",
+    "aws_cdk-lib>=2.51.1"
```

Comment:
Any particular reason you used 2.51.1? From the perspective of this handler, all you really care is that it's possible to synthesize the construct. We probably would want to be very flexible with the versions we allow on this.